static/js/pages/certificates/BrowserImport.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC, useState } from "react";
+import { Col, Row, Tabs } from "@canonical/react-components";
+
+const TABS: string[] = ["Firefox", "Chrome"];
+
+const BrowserImport: FC = () => {
+  const [activeTab, handleTabChange] = useState("firefox");
+
+  return (
+    <Row>
+      <Col size={8}>
+        <Tabs
+          links={TABS.map((tab) => ({
+            label: tab,
+            active:
+              tab.toLowerCase() === activeTab ||
+              (tab === "firefox" && !activeTab),
+            onClick: () => handleTabChange(tab.toLowerCase()),
+          }))}
+        />
+
+        {activeTab === "firefox" && (
+          <div tabIndex={0} role="tabpanel" aria-labelledby="firefox">
+            <ul className="p-list--divided">
+              <li className="p-list__item">
+                Paste this link into the address bar{" "}
+                <a
+                  href="about:preferences#privacy"
```

Comment:
This link (and the one for Chrome as well) doesn't seem to work, so maybe it should be removed?